docker/rel2/dockit

```diff
@@ -1,5 +1,7 @@
-#!/bin/sh -eu
+#!/bin/bash
```

Comment:
using bash instead because i can't figure out how get pipefail working in sh

Reply:
I'm not sure what pipefail is. Let me look at what's going on here...

But you want -eu. It helps find bugs in shell scripts. -e says exit on first error. -u says exit on using an undefined variable (instead of replacing it with "").

- [ ]

Reply:
The docker script isn't using tee. What's the problem again?